modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipFormBase.tsx

```diff
@@ -165,6 +170,23 @@ export function ObjectRelationshipFormBase({
 		[objectDefinitions, values.objectDefinitionId1, values.type]
 	);
 
+	const filteredRelationships = useMemo(() => {
+		if (Liferay.FeatureFlags['LPS-158478']) {
+			return objectDefinitions.filter(({label}) => {
+				return label[defaultLanguageId]
+					?.toLocaleLowerCase()
+					.includes(query.toLowerCase());
+			});
+		}
+		else {
+			return filteredObjectDefinitions.filter(({label}) => {
+				return label[defaultLanguageId]
+					?.toLocaleLowerCase()
+					.includes(query.toLowerCase());
+			});
+		}
+	}, [objectDefinitions, filteredObjectDefinitions, query]);
```

Comment:
I think these should be in alphabetical order, like `[filteredObjectDefinitions, objectDefinitions, query]`

Reply:
Done! Thanks!